package.json

```
@@ -1,95 +1,19 @@
 {
-  "name": "tinymce-ember",
-  "version": "3.0.2",
-  "description": "TinyMCE Ember addon",
-  "keywords": [
-    "ember-addon",
-    "tinymce",
-    "editor"
-  ],
-  "repository": "https://github.com/concordnow/tinymce-ember",
-  "license": "MIT",
-  "author": "Concord Worlwide Inc.",
-  "directories": {
-    "doc": "doc",
-    "test": "tests"
-  },
+  "private": true,
+  "workspaces": ["addon", "test-app"],
+
   "scripts": {
-    "build": "ember build --environment=production",
-    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
-    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
-    "lint:hbs": "ember-template-lint .",
-    "lint:hbs:fix": "ember-template-lint . --fix",
-    "lint:js": "eslint . --cache",
-    "lint:js:fix": "eslint . --fix",
+    "prepare": "cd addon && yarn build",
     "release": "release-it",
-    "start": "ember serve",
-    "test": "npm-run-all lint test:*",
-    "test:ember": "ember test",
-    "test:ember-compatibility": "ember try:each"
+    "test": "cd test-app && yarn test:ember"
```

Shouldn't you use `yarn workspace *` commands ?
 
```suggestion
    "test": "yarn workspace test-app test:ember"
```

---

I try to avoid yarn specific command as we might want to change package manager